src/ceph-daemon

```diff
@@ -1096,6 +1100,25 @@ def command_bootstrap():
         logger.info('Adding host %s...' % host)
         cli(['orchestrator', 'host', 'add', host])
 
+    if not args.skip_dashboard:
+        logger.info('Enabling the dashboard module...')
+        cli(['mgr', 'module', 'enable', 'dashboard'])
+        logger.info('Generating a dashboard self-signed certificate...')
+        cli(['dashboard', 'create-self-signed-cert'])
```

Comment:
AFAIL, @tserong did run into a race here. I guess we have to wait, till the module is successfully enabled?

Reply:
yep, fixed